plugins/schedule/base/lib/iCal/kSchedulingICalEvent.php

```diff
@@ -296,20 +296,17 @@ public static function fromObject(KalturaScheduleEvent $event)
 			{
 				if ($resourceIds)
 				{
-					$resourcesNames = array();
 					$c = new Criteria();
-					$c->add(ScheduleResourcePeer::PARTNER_ID, $event->partnerId);
-					foreach ($resourceIds as $resourceId)
+					$c->add(ScheduleResourcePeer::PARTNER_ID, 102);

```

Comment:
??

Reply:
oops! i was testing it with a script on my system